dask/dataframe/io/tests/test_parquet.py

```diff
@@ -369,3 +390,17 @@ def test_timestamp_index():
         ddf.to_parquet(fn)
         ddf2 = dd.read_parquet(fn)
         assert_eq(ddf, ddf2)
+
+
+@pytest.mark.skipif(not pyarrow, reason='pyarrow not found')
+@pytest.mark.xfail(reason="to_parquet does not write nulls by default")
```

Comment:
Perhaps we can remove this xfail when making the change to the argument in Dask as proof that it's doing the expected thing.